packages/shared/src/components/post/PostComments.tsx

```diff
@@ -67,6 +68,9 @@ export function PostComments({
         refetchOnWindowFocus: false,
       },
     );
+
+  useCommentContentPreferenceMutationSubscription({ queryKey });
```

Comment:
This does start to feel a bit dangerous already.
Eg companion comments are now not accounted for (not that, that specific case is a big issue, but just saying with manually having to mutate other stuff it becomes quickly easy to forget a spot or not do it.

Reply:
I have to double check companion still yes, but it is using the same component, probably issue is that it does not use QueryClient 🙈 

I would not say its dangerous but introduces a mindset shift where we think about invalidation as separate component which we need to take care of when building components and if components need it. Muck like state, styling etc. Would love to go deeper into this approach and discuss pros/cons.

Reply:
I remove follow option on comments in companion for now, will recheck it at the end. Extension works.